api/routes/__init__.py

```diff
@@ -1,17 +1,22 @@
 from flask import Flask
 from flask_cors import CORS
-from api.routes import api_news, api_games, api_sports
+from api.routes import api_news, api_games, api_sports, api_teams
+from api.routes.cache import cache
 
 # Ініціалізація додатку
 def create_app():
     app = Flask(__name__)
     CORS(app)
+    app.config['CACHE_TYPE'] = 'SimpleCache'
+    app.config['CACHE_DEFAULT_TIMEOUT'] = 300
```

Comment:
What are those? seconds? please specify.

If those are 5 minutes, I'd suggest writing 60 * 5 and writing a comment "5 min" or smth